### PR TITLE
fix CORS issue in production deployment

### PR DESCRIPTION
**Pull request by mjaquiery**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -8,6 +8,8 @@ GALVANALYSER_HARVESTER_TEST_PATH=./.run/test_datafiles
 # Required to get react running:
 NODE_OPTIONS=--openssl-legacy-provider
 
+FRONTEND_PORT=80
```
Comment (martinjrobins) [inline .env:11]: should specify the entire hostname, not just the port?
`HOSTNAME=localhost:80`

**Review by martinjrobins — CHANGES_REQUESTED** [review]
I would suggest that we allow the user to specify the entire hostname

**Comment by mjaquiery** [comment]
@martinjrobins This version should be better. Any chance you can test it on the Azure setup and verify it works? I can run it successfully without CORS issues in dev/prod mode quite happily, and even specify a port for it to run on. As is, it'll run on port 80 in both dev and prod modes. 

As a side-effect, I've removed nginx's proxying of the django server and just allowed direct access because that container exposes its http interface over port 5000, so you've got http://localhost for the frontend and http://localhost:5000 for the backend.

**Review by martinjrobins — APPROVED** [review]
looks great, thanks @mjaquiery !

**Pull request merged**